npm/src/db/redis.ts

```diff
@@ -96,7 +96,17 @@ class Redis implements DatabaseDriver {
       tx = tx.sAdd(dbutils.keyFromParts(dbutils.indexPrefix, idxKey), key);
       tx = tx.sAdd(dbutils.keyFromParts(dbutils.indexPrefix, k), idxKey);
     }
-
+    const timestamp = Number(Date.now());
+    const negetiveTimestamp = -Math.abs(timestamp);
```

Comment:
`negativeTimestamp`. also let's add a comment here saying why we are using negative timestamps